ops/pebble.py

```diff
@@ -1496,6 +1507,9 @@ class NoticeType(enum.Enum):
     notices is the change ID.
     """
 
+    UNKNOWN = 'unknown'
+    """Used if we receive an unrecognised notice type (e.g. from future Pebble versions)."""
```

Comment:
Nit: I think in docs its better to spell out these abbreviations. For one thing, the difference between "e.g." and "i.e." can be confusing (e.g. => for example, i.e. => that is).

```suggestion
    """Used if we receive an unrecognised notice type (for example, from future Pebble versions)."""
```